client/src/actions/index.js

```diff
@@ -0,0 +1,13 @@
+export const changeView = (view) => {
+  return {
+    type: 'CHANGE_VIEW',
+    view
```

Comment:
Independent of this pull request, but how do you feel about using [Flux Standard Actions](https://github.com/acdlite/flux-standard-action) format going forward?